### ISSUE TITLE
Obihai - user and password

### ISSUE THREAD

**Issue by felafel** [home-assistant/home-assistant.github.io#27408]
### Feedback

I have tried my email login and "admin" with the password I use to manage the device on the Obitalk web page.  I get "invalid authentication".  What should I use and do I need to set something else up?  The internal IP address for the device looks to be correct.

### URL

https://www.home-assistant.io/integrations/obihai/

### Version

2023.5.3

**Comment by ejpenney** [comment]
This should be the local device login information, not your obitalk credentials.  This integration has nothing to do with ObiTalk.  Default your username is "admin" and password is "admin".  It should be whatever you use to login when you visit the device via your browser (EG: http://<ip_address>)

**Comment by felafel** [comment]
Thanks for the quick reply.

The issue is that the device is plugged into my local network using the WAN connection.

This is the solution if you are set up this way.
http://www.obitalk.com/forum/index.php?topic=3108.msg20765#msg20765

TL;DR
- Dial ***0 from the phone connected to the OBi202
- Enter 30#
- Press 1 to Enter a New Value
- Press 1# to Enable
- Press 1 to Save

Thanks again
